### ISSUE TITLE
Sounds not playing

### ISSUE THREAD

**Issue by L4D15** [otigon/automated-jb2a-animations#307]
I'm trying to play a sound when an animation is triggered but I get the following error and no sound is played:

```
foundry.js:63059 Uncaught (in promise) DOMException: The play() request was interrupted by a call to pause(). https://goo.gl/LdLk22
```

I tried setting delay and no-delay, but the error takes place in both cases.

![Captura de pantalla 2022-01-05 100112](https://user-images.githubusercontent.com/2948600/148190154-a8d62d7d-4e63-467e-9fbf-368f877cc305.png)

I'm on Foundry 9 build 238, Automate Animations 1.1.3, running on The Forge.

**Comment by otigon** [comment]
@Haxxer , have you ever run across an error like this when working with sounds?

**Comment by Haxxer** [comment]
This may be because the browser was not in focus during the time it was attempted to be played. Modern browsers are very picky about sounds, specifically sounds playing in different tabs - the browser straight up stops sounds from playing on windows or tabs that aren't in focus.

**Comment by L4D15** [comment]
The browser and the tab are on focus, so mush so that I'm triggering the animation after placing a template on the scene for a dragon's breath weapon, I am interacting with Foundry when the animation triggers and starts playing, but the sound does not play and the reported error pops in the console with the following link https://developers.google.com/web/updates/2017/06/play-request-was-interrupted

**Comment by L4D15** [comment]
I've been fiddleing around different scenarios and found that, if there is music playing in the background, the sound does not play, but the error don't show up in the console. I tried playing the sound from the Soundboard (Blitz Soundboard module) and it plays just fine.

Finally, I tried assigning another file, it didn't play either, and I assigned the original file back again, reloaded with Ctrl + F5, and now it plays correctly with the animation. Not sure what was causing the error but somehow is now fixed.
